lib/player.js

```diff
@@ -876,51 +766,14 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     // If LCEVC Decoder exists close it.
     this.closeLcevcDec_();
 
+    const destroyPromise = this.detach();
```

Comment:
Might be more accurately named detachPromise

Reply:
Done.